packages/dashboard/src/components/TokenIcon/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { makeStyles, Avatar, Theme, AvatarProps } from '@material-ui/core'
+import {
+    CONSTANTS,
+    useBlockie,
+    useChainDetailed,
+    ChainId,
+    constantOfChain,
+    isSameAddress,
+    useConstantNext,
+    TOKEN_CONSTANTS,
+} from '@masknet/web3-shared'
+import { useImageFailover } from '../TokenList/useImageFailover'
+import SPECIAL_ICON_LIST from './TokenIconSpeialIconList.json'
+import { useStylesExtends } from '@masknet/shared'
+import { formatEthereumAddress } from '@masknet/shared'
+
+//#region fix icon image
+function resolveTokenIconURLs(address: string, baseURIs: string[], chainId: ChainId, logoURI?: string) {
+    const checkSummedAddress = formatEthereumAddress(address)
+
+    if (isSameAddress(constantOfChain(TOKEN_CONSTANTS, chainId).NATIVE_TOKEN_ADDRESS, checkSummedAddress)) {
+        return baseURIs.map((x) => `${x}/info/logo.png`)
+    }
+
+    const specialIcon = SPECIAL_ICON_LIST.find((item) => item.address === address)
+
+    if (specialIcon) return [specialIcon.logo_url]
+
+    const fullIconAssetURIs = baseURIs.map((x) => `${x}/assets/${checkSummedAddress}/logo.png`)
+    return logoURI ? [logoURI] : fullIconAssetURIs
+}
+//#endregion
+
+const useStyles = makeStyles((theme: Theme) => ({
+    icon: {
+        width: 36,
+        height: 36,
+        backgroundColor: theme.palette.common.white,
```

Comment:
white background in both light and dark mode?

Reply:
I tested on local. It's ok for token icon.